tests/testsuite/profile_targets.rs

```diff
@@ -185,17 +193,17 @@ fn profile_selection_build_all_targets() {
     //   bin      dev        build
     //   example  dev        build
     p.cargo("build --all-targets -vv")
-        .with_stderr_unordered("\
+        .with_stderr_data(str![[r#"
 [LOCKING] 3 packages to latest compatible versions
-[COMPILING] bar [..]
-[RUNNING] `[..] rustc --crate-name bar --edition=2015 bar/src/lib.rs [..]--crate-type lib --emit=[..]link[..]-C codegen-units=1 -C debuginfo=2 [..]
-[RUNNING] `[..] rustc --crate-name bar --edition=2015 bar/src/lib.rs [..]--crate-type lib --emit=[..]link -C panic=abort[..]-C codegen-units=1 -C debuginfo=2 [..]
-[RUNNING] `[..] rustc --crate-name bar --edition=2015 bar/src/lib.rs [..]--crate-type lib --emit=[..]link[..]-C codegen-units=5 [..]
-[COMPILING] bdep [..]
-[RUNNING] `[..] rustc --crate-name bdep --edition=2015 bdep/src/lib.rs [..]--crate-type lib --emit=[..]link[..]-C codegen-units=5 [..]
-[COMPILING] foo [..]
-[RUNNING] `[..] rustc --crate-name build_script_build --edition=2015 build.rs [..]--crate-type bin --emit=[..]link[..]-C codegen-units=5 [..]
-[RUNNING] `[..]/target/debug/build/foo-[..]/build-script-build`
+[COMPILING] bar v0.0.1 ([ROOT]/foo/bar)
+[RUNNING] `[..] rustc --crate-name bar --edition=2015 bar/src/lib.rs [..]--crate-type lib --emit=[..]link -C embed-bitcode=[..]-C codegen-units=1 -C debuginfo=2 [..]`
```

Comment:
Oh, I forgot to mention this! I'm having a hard time with an old assertion here that seems to be flaky.

My best guess is that an assertion without `panic-abort` is a superset of the one with `panic-abort`. This could lead to a mismatch when the running command **with** `panic-abort` executes first and consumes the assertion **without** `panic-abort`. As a result, the command **without** `panic-abort` wouldn't be able to find a valid assertion afterwards.

Reply:
It still seems a bit odd to me using `-C embed-bitcode` as a workaround, though it doesn't really too bad so fine with it.